layouts/shortcodes/install-python.html

```diff
@@ -1,6 +1,6 @@
 <p class="mt-4">
-    Install <strong><a href="https://www.python.org/downloads/" target="_blank" rel="noopener">Python
-            version 3.6 or later</a></strong>.
+    Install <a href="https://www.python.org/downloads/" target="_blank" rel="noopener">Python
+            version 3.6 or later</a>. To reduce potential issues with setting up your Python environment on Windows or macOS, you should install Python through the official Python installer.
```

Comment:
Should we provide a link under "official Python installer"?  I assume this means the python.org distribution?

Reply:
Correct. People can get to the official installer for their OS through the link in the previous sentence.

Reply:
Ah - got it.